src/Adapter/Customer/CommandHandler/AddCustomerHandler.php

```diff
@@ -69,12 +69,13 @@ public function __construct(Hashing $hashing, $legacyCookieKey)
      */
     public function handle(AddCustomerCommand $command)
     {
+        // Prepare new legacy customer object
         $customer = new Customer();
 
+        // Load it up with the data from the command
         $this->fillCustomerWithCommandData($customer, $command);
 
-        // validateFieldsRequiredDatabase() below is using $_POST
-        // to check if required fields are set
+        // validateFieldsRequiredDatabase() below is using $_POST to check if required fields are set
         $_POST[RequiredField::PARTNER_OFFERS] = $command->isPartnerOffersSubscribed();
```

Comment:
CommandHandler MUST NOT be coupled with request. (also we never use $_POST or $_GET in symfony context, there is symfony Request object for cleaner way of getting request params).

Reply:
With that said, this field must be probably added in your form, and passed in formDataHandler. FormData handler is responsible to handle the form data (request data) and convert it to related command.

Reply:
Julius, I did not add this code, we can do it in another PR. I don't to increase the scope million times again and never merge it.

Reply:
Oh right, I just saw it wasn't your change